spec/domain/services/fetch_course_events/service_spec.rb

```diff
@@ -261,22 +261,74 @@
     end
 
     context 'update_course_active_dates events' do
-      let(:current_time)       { Time.current }
-      let(:starts_at)          { (current_time - 1.day).iso8601 }
-      let(:ends_at)            { (current_time + 1.day).iso8601 }
+      let(:current_time)       { DateTime.iso8601(Time.current.iso8601) }
```

Comment:
I think you could use `DateTime.current.iso8601` here

Reply:
Need to convert it back and forth to normalize it (i.e. remove sub-second precision). Otherwise the comparison fails.